<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
<testsuite skipped="6" hostname="QTLSYS1-182" name="tests.QCStore" tests="8" failures="0" timestamp="26 Mar 2019 13:00:03 GMT" time="1577.001" errors="0">
  <testcase name="JQC_AgeStore_Deposit_Return_Refinance_Void_Txn" time="841.321" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Refinance_Rescind_Txn" time="735.680" classname="tests.QCStore"/>
=======
<testsuite skipped="6" hostname="QTLSYS2-48" name="tests.QCStore" tests="7" failures="0" timestamp="02 Apr 2019 05:27:44 GMT" time="806.719" errors="0">
  <testcase name="RefinanceStepup" time="806.719" classname="tests.QCStore"/>
>>>>>>> anoop1.1
=======
<testsuite skipped="6" hostname="QTLSYS2-238" name="tests.QCStore" tests="28" failures="3" timestamp="01 Apr 2019 19:09:40 GMT" time="22403.674" errors="0">
  <testcase name="QC_NewLoan_Promotion_Txn" time="377.401" classname="tests.QCStore"/>
  <testcase name="QC_RefinanceStepDown_Txn" time="1133.943" classname="tests.QCStore"/>
  <testcase name="QC_RefinanceStepDown_Void_Txn" time="1388.917" classname="tests.QCStore"/>
  <testcase name="QC_OpenLoan_NewLoan_Txn" time="444.370" classname="tests.QCStore"/>
  <testcase name="QC_NewLoan_Void_Txn" time="326.327" classname="tests.QCStore"/>
  <testcase name="QC_AgeStore_Void_Txn" time="506.256" classname="tests.QCStore"/>
  <testcase name="QC_Deposit_Menu_Txn" time="740.289" classname="tests.QCStore"/>
  <testcase name="QC_Deposit_GraceDays_Txn" time="834.424" classname="tests.QCStore"/>
  <testcase name="QC_FutureDeposit_DropDown" time="1052.794" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Deposit_Void_Txn" time="1231.775" classname="tests.QCStore"/>
  <testcase name="QC_Epp_PrePayment_Txn" time="1350.343" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Check_Deposit_Clear_Txn" time="1552.275" classname="tests.QCStore"/>
  <testcase name="QC_EppDepositClear_Refund_Txn" time="1510.456" classname="tests.QCStore"/>
  <testcase name="QC_Epp_PrePayment_Return_Txn" time="1351.038" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Return_NSFPayment_Txn" time="1432.047" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Return_ReDeposit_Txn" time="1413.732" classname="tests.QCStore"/>
  <testcase name="QC_Void_FullReturnPayment_Txn" time="1389.679" classname="tests.QCStore"/>
  <testcase name="QC_PartialPayment_RefundVoid_Txn" time="1107.442" classname="tests.QCStore"/>
  <testcase name="QC_Refinance_StepSameURP_Txn" time="851.613" classname="tests.QCStore">
=======
<testsuite skipped="6" hostname="QTLSYS2-238" name="tests.QCStore" tests="8" failures="1" timestamp="08 Apr 2019 04:37:50 GMT" time="833.716" errors="0">
  <testcase name="QC_GraceDays_EODDeposit_Txn" time="506.734" classname="tests.QCStore"/>
  <testcase name="QC_FutureDeposit_EOD_Txn" time="326.982" classname="tests.QCStore">
>>>>>>> brahmith1.1
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
<<<<<<< HEAD
at tests.QCRefinance.refinanceStepSame(QCRefinance.java:730)
at tests.QCStore.QC_Refinance_StepSameURP_Txn(QCStore.java:1457)
=======
<testsuite skipped="6" hostname="QTLSYS2-48" name="tests.QCStore" tests="7" failures="0" timestamp="02 Apr 2019 06:56:00 GMT" time="14.941" errors="1">
  <testcase name="RefinanceStepup" time="14.941" classname="tests.QCStore">
    <error type="org.openqa.selenium.StaleElementReferenceException" message="Element is no longer valid (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &#039;3.14.0&#039;, revision: &#039;aacccce0&#039;, time: &#039;2018-08-02T20:19:58.91Z&#039;
System info: host: &#039;QTLSYS2-48&#039;, ip: &#039;192.168.2.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_171&#039;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserAttachTimeout: 0, browserName: internet explorer, cssSelectorsEnabled: true, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, handlesAlerts: true, ie.browserCommandLineSwitches: , ie.ensureCleanSession: false, ie.forceCreateProcessApi: false, ie.usePerProcessProxy: false, ignoreProtectedModeSettings: false, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:15875/, javascriptEnabled: true, nativeEvents: true, platform: WINDOWS, platformName: WINDOWS, requireWindowFocus: false, takesScreenshot: true, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: 843ed441-09c4-459f-be01-cf26c19f1035">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element is no longer valid (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'QTLSYS2-48', ip: '192.168.2.48', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserAttachTimeout: 0, browserName: internet explorer, cssSelectorsEnabled: true, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, handlesAlerts: true, ie.browserCommandLineSwitches: , ie.ensureCleanSession: false, ie.forceCreateProcessApi: false, ie.usePerProcessProxy: false, ignoreProtectedModeSettings: false, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:15875/, javascriptEnabled: true, nativeEvents: true, platform: WINDOWS, platformName: WINDOWS, requireWindowFocus: false, takesScreenshot: true, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: 843ed441-09c4-459f-be01-cf26c19f1035
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
at tests.ACSRBorrowerRegistration.borrowerReg(ACSRBorrowerRegistration.java:117)
at tests.QCStore.RefinanceStepup(QCStore.java:2491)
>>>>>>> anoop1.1
=======
at tests.QCBorrowerRegistration.borrowerRegistration(QCBorrowerRegistration.java:519)
at tests.QCStore.QC_FutureDeposit_EOD_Txn(QCStore.java:1706)
>>>>>>> brahmith1.1
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
<<<<<<< HEAD
    </failure>
<<<<<<< HEAD
  </testcase> <!-- QC_Refinance_StepSameURP_Txn -->
  <testcase name="QC_Refinance_StepSameVoidURP_Txn" time="735.355" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.QCRefinance.refinanceStepSame(QCRefinance.java:730)
at tests.QCStore.QC_Refinance_StepSameVoidURP_Txn(QCStore.java:1516)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- QC_Refinance_StepSameVoidURP_Txn -->
  <testcase name="QC_Refinance_StepDownURP_Txn" time="904.619" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.QCRefinance.refinanceStepDown(QCRefinance.java:359)
at tests.QCStore.QC_Refinance_StepDownURP_Txn(QCStore.java:1331)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- QC_Refinance_StepDownURP_Txn -->
  <testcase name="QC_Refinance_StepDownVoidURP_Txn" time="768.578" classname="tests.QCStore"/>
>>>>>>> brahmith1.1
=======
    </error>
  </testcase> <!-- RefinanceStepup -->
>>>>>>> anoop1.1
=======
<testsuite skipped="6" hostname="QTLSYS1-182" name="tests.QCStore" tests="28" failures="0" timestamp="29 Mar 2019 18:23:22 GMT" time="17519.504" errors="0">
  <testcase name="JQC_AgeStore_Deposit_PrePayment_Txn" time="721.353" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_PrePayment_Void_Txn" time="754.812" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_ClearDropdown_Txn" time="604.700" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_ClearMenu_Txn" time="722.352" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_AutoClear_Txn" time="805.171" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_PrePayment_Return_Txn" time="811.201" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_PrePayment_Clear_Txn" time="864.832" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_NSFPosting_Refinance_Rescind_Txn" time="752.614" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_ACHReturn_Refinance_Rescind_Txn" time="830.666" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_Return_Refinance_Txn" time="749.239" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Deposit_Return_Refinance_Void_Txn" time="776.633" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Refinance_Rescind_Txn" time="703.861" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Redeposit_Prepayment_Void_Prepayment_Return_Txn" time="930.123" classname="tests.QCStore"/>
  <testcase name="JQC_Return_ReDeposit_Clear_Txn" time="912.469" classname="tests.QCStore"/>
  <testcase name="JQC_BuyBack_Txn_Testdata" time="579.185" classname="tests.QCStore"/>
  <testcase name="JQC_BuyBack_Void_Txn_Testdata" time="697.552" classname="tests.QCStore"/>
  <testcase name="JQC_BuyBack_Rebate_Txn_Testdata" time="395.264" classname="tests.QCStore"/>
  <testcase name="JQC_BuyBack_Rebate_Void_Txn_Testdata" time="440.359" classname="tests.QCStore"/>
  <testcase name="JQC_ChkDeposit_RtnChk_ReDeposit_Payment_Void_Clear_Txn_Testdata" time="907.026" classname="tests.QCStore"/>
  <testcase name="JQC_Deposit_Return_EPP_Txn_Testdata" time="1004.774" classname="tests.QCStore"/>
  <testcase name="JQC_EPP_AgestoreInstallmentwise_EPPPayment_Txn_Testdata" time="1942.540" classname="tests.QCStore"/>
  <testcase name="JQC_EPP_Txn_Testdata" time="612.777" classname="tests.QCStore"/>
>>>>>>> janaki1.2
=======
  </testcase> <!-- QC_FutureDeposit_EOD_Txn -->
>>>>>>> brahmith1.1
=======
<testsuite skipped="6" hostname="QTLSYS1-182" name="tests.QCStore" tests="7" failures="0" timestamp="04 Apr 2019 05:52:23 GMT" time="735.365" errors="0">
  <testcase name="JQC_PartialWORecovery_FullWORecovery_Void_Txn_Testdata" time="735.365" classname="tests.QCStore"/>
>>>>>>> janaki1.2
  <testcase name="JQC_AgeStore_GraceDays_DCDeposit_Menu_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_GraceDays_DCDeposit_Menu_Txn -->
  <testcase name="JQC_ACH_PartialWORecovery_FullWORecovery_Void_Txn_Testdata" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_ACH_PartialWORecovery_FullWORecovery_Void_Txn_Testdata -->
  <testcase name="JQC_AgeStore_GraceDays_DCDeposit_Dropdown_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_GraceDays_DCDeposit_Dropdown_Txn -->
  <testcase name="JQC_AgeStore_DCDeposit_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_DCDeposit_Txn -->
  <testcase name="JQC_NewLoan_Veritec_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_NewLoan_Veritec_Txn -->
  <testcase name="JQC_AgeStore_DCDeposit_Menu_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_DCDeposit_Menu_Txn -->
</testsuite> <!-- tests.QCStore -->
